### DISCUSSION TITLE
Kubernetes Auto Complete

### DISCUSSION THREAD

**Discussion by Azaad78** [redhat-developer/vscode-yaml#473] [Q&A]
How add more items to Kubernetes Auto Complete?

**Comment by gorkem** [comment]
What do you mean by more items? CRDs?

**Reply by rawkode** [reply]
@gorkem, Can we use some comment pragma to indicate that a YAML file can be validated against a CRD.yaml?

**Reply by gorkem** [reply]
No. We only have basic support for K8s schema. I am also reluctant to add more kubernetes specific features since we started to provide extension APIs, some of the other more specialized extensions have been able to provide a better match than we do. 

As an alternate, we could think of a more generic solution like a JSON path to the schema mapping in addition to current mechanisms but that is not on our immediate plans. However, we do accept PRs :)
